ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op_width_sharded_program_factory.cpp

```diff
@@ -787,11 +791,11 @@ operation::ProgramWithCallbacks multi_core_optimized_conv_width_sharded_v2_impl(
         );
 
         SetRuntimeArgs(program, act_kernel_id, CoreCoord(core_x, core_y), rt_args);
-
         SetRuntimeArgs(program, weights_kernel_id, CoreCoord(core_x, core_y), {
             core_index * weight_block_w_ntiles,
             b.buffer()->address(),
-            bias_base_address
+            bias_base_address,
+            (uint32_t) (core_index < output_num_cores ? 1 : 0)
```

Comment:
Why have `? 1 : 0`? Doesn't a bool map to uint32_t properly?

Reply:
Fixed.